package.json

```diff
@@ -27,15 +29,18 @@
     "husky": "^1.3.1",
     "marked": "^0.6.1",
     "mocha": "^6.0.1",
-    "semantic-release": "^15.13.3"
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^2.6.3",
+    "semantic-release": "^15.13.3",
+    "typescript": "^3.3.4000"
```

Comment:
What is typescript dependency for?

Reply:
TypeScript is needed by the TypeScript parse, which is needed to check TypeScript-specific syntax like the parameter properties.

Reply:
@rasenplanscher , would you mind rebasing your changes onto `master`?